grails-app/controllers/au/org/ala/profile/StatisticsController.groovy

```diff
@@ -0,0 +1,57 @@
+package au.org.ala.profile
+
+import au.org.ala.profile.security.Role
+import grails.converters.JSON
+
+class StatisticsController extends BaseController {
+
+	StatisticsService statisticsService
+
+	def index() {
+		if (!params.opusId) {
+			badRequest "opusId is a required parameter"
+		} else {
+			Opus opus = getOpus()
+
+			if (!opus) {
+				notFound "No opus found for ${params.opusId}"
+			} else {
+				def statistics = []
+
+				int profileCount = Profile.count
+				statistics.add([
+				        name: 'Profiles',
+						value: profileCount
+				])
+
+				int editorCount = opus.authorities.count {
+					eq("role", Role.ROLE_PROFILE_EDITOR)
+				}
+				statistics.add([
+				        name: 'Editors',
+						value: editorCount
+				])
+
+				Profile profile = statisticsService.lastEditedProfile(opus)
+				statistics.add([
+				        name: 'Last Edited',
+						value: "${profile.scientificName} by ${profile.lastUpdatedBy} on ${profile.lastUpdated.format("dd MMM yyyy")}"
+				])
+
+				int namesNotInNSL = statisticsService.profilesWithNameNotInNSL(opus)
```

Comment:
You can just inject the ReportService into the StatisticController.

There is similarity in the roles, but not enough to merge them, in my opinion. Keeping the main business logic in services and injecting them where necessary is a better approach.